### PR TITLE
[MLOP-633] Butterfree dev workflow, update documentation

### PR DESCRIPTION
**Pull request by moromimay**
## Why? :open_book:
Update documentation to inform publicly the dev workflow.

## What? :wrench:
- Workflow.md

## Type of change
- [x] Documentation

## Checklist
- [x] I have made corresponding changes to the documentation;
- [x] Add labels to distinguish the type of pull request. Available labels are `bug`, `enhancement`, `feature`, and `review`.

### CODE REVIEW

**Review by roelschr — DISMISSED** [review]

**Review by acsilveira — APPROVED** [review]
LGTM!

File: WORKFLOW.md [hunk 1]
```diff
@@ -2,20 +2,18 @@
 
 ## Features
 
-A feature is based on the `master` branch and merged back into the `master` branch.
-
-![](https://docs.microsoft.com/en-us/azure/devops/repos/git/media/branching-guidance/featurebranching.png?view=azure-devops)
+A feature is based on the `staging` branch and merged back into the `stagimg` branch.
```
Comment (acsilveira) [inline WORKFLOW.md:5]: Just a typo here 'stagimg' > 'staging'.

**Review by acsilveira — APPROVED** [review]
LGTM.

**Review by acsilveira — APPROVED** [review]
LGTM.

**Pull request merged**